Automattic/wp-calypso
# Gutenberg: Use primitives for Publicize icons

As @sirreal suggested in https://github.com/Automattic/wp-calypso/pull/29317#discussion_r240952264 this PR updates the service icons in our Publicize extension for Gutenberg to use the Gutenberg primitives. While this introduces no changes in the end results, it will ensure that any enhancement that these primitives get in the future will reflect our icons as well.

#### Changes proposed in this Pull Request

* Use the Gutenberg primitives instead of pure SVG elements

#### Testing instructions

* Spin up calypso.live from the link below.
* Select a simple WP.com site.
* Start a post.
* Click the Jetpack icon.
* Connect at one of each services to your site.
* Verify markup of icons is unchanged and they still look well.
